pkg/services/auth/auth_token.go

```diff
@@ -55,7 +55,14 @@ func (s *UserAuthTokenService) ActiveTokenCount(ctx context.Context) (int64, err
 }
 
 func (s *UserAuthTokenService) CreateToken(ctx context.Context, userId int64, clientIP, userAgent string) (*models.UserToken, error) {
-	clientIP = util.ParseIPAddress(clientIP)
+	s.log.Debug("Creating user auth token", "clientIp", clientIP)
+	clientIP, err := util.ParseIPAddress(clientIP)
+	if err != nil {
+		s.log.Debug("Failed to parse client IP address", "clientIp", clientIP, "err", err)
+		clientIP = ""
```

Comment:
Assuming that we want to be tolerant here and just go with an empty client IP address.

Reply:
Makes sense :+1: Just remove the debug log messages - I guess you used them for manual tests?

Reply:
@marefr I thought the debug messages might be useful to keep around. You think they add to much noise?

Reply:
Yeah given that we already have debug message on [line 99 below](https://github.com/grafana/grafana/pull/19573/files#diff-df4f6a94ac93c2556bf59f514f4c847eR99). 

At least if there's no problem parsing the IP address we don't need to debug log (line 63-65 below).

If there are problem parsing the IP address we can keep the debug log, but since `clientIP` are or may be overwritten when calling `clientIP, err := util.ParseIPAddress(clientIP)` I'm not sure if the debug log message will include the source IP address or an empty string?

Reply:
@marefr You're right, the logging of `clientIP` didn't make sense since it was already reassigned. I renamed the parameter to `clientAddr` since I think this makes more sense anyway, as it may contain a port section. Also removed the offending logs.

Reply:
Great :+1: